demo/prepare_xpack.py

```diff
@@ -0,0 +1,46 @@
+from elasticsearch_xpack import XPackClient
+import getpass
+import sys
+from utils import Elasticsearch
+
+if __name__ == "__main__":
+    if len(sys.argv) == 2:
+        password = getpass.getpass()
+    elif len(sys.argv) == 3:
+        password = sys.argv[2]
+    else:
+        sys.exit(-1)
```

Comment:
some error doc would be good here, perhaps just print(__doc__) before the sys.exit and add a very brief usage as a """foo bar baz""" at the top of the file. Probably should have done that in my first patch ...